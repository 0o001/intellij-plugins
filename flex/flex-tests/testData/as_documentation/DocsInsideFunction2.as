package {
public class DocsInsideFunction2 {

  [Bindable("change")]

      /**
       *  Gets the value of a specified resource as a Class.
       *  @return The resource value, as a <code>Class</code>,
       *  or <code>null</code> if it is not found.
       */
      function getC<caret>lass(bundleName:String, resourceName:String,
                        locale:String = null):Class;


}
}