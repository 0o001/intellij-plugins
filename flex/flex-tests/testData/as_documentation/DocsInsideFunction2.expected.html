<div class='definition'><pre>DocsInsideFunction2.getClass(<br>    bundleName: String,<br>    resourceName: String,<br>    locale?: String = null): <a href="psi_element://Class">Class</a></pre></div><div class='content'><p>Gets the value of a specified resource as a Class.</div><table class='sections'><tr><td valign='top' class='section'><p>Returns:</td><td valign='top'><p>The resource value, as a <code>Class</code>,
<p>or <code>null</code> if it is not found.</td></table>